src/main/java/org/jabref/logic/importer/AuthorListParser.java

```diff
@@ -96,6 +97,9 @@ private static StringBuilder buildWithAffix(Collection<Integer> indexArray, List
     public AuthorList parse(String listOfNames) {
         Objects.requireNonNull(listOfNames);
 
+        // Handle the statement "and others" at the end, removing it
+        listOfNames = StringUtils.removeEndIgnoreCase(listOfNames.trim(), " AND OTHERS");
```

Comment:
Can you move this StringUtils  method to JabRef's StringUtil class located under model?
And then annotate it with  @ApacheCommonsLang3Allowed()
The reason is that we want to have it located in one place

Reply:
Done!